src/stylesheets/core/_margin-padding.scss

```diff
@@ -0,0 +1,114 @@
+    
+     %padding-bottom-#{$i}x,
+     %padding-vertical-#{$i}x{ padding-bottom: $spacing * $i !important; }
+    
+     %padding-left-#{$i}x,
+     %padding-horizontal-#{$i}x { padding-left: $spacing * $i !important; }
+    
+     %padding-right-#{$i}x,
+     %padding-horizontal-#{$i}x{ padding-right: $spacing * $i !important; }
+    }
+}
+
+$spacing: 5px;
+@include margin-and-padding-spacing($spacing);
+
+.m_All-3x { @extend %margin-3x; }
+.m_All-4x { @extend %margin-4x; }
+.m_V-2x {  @extend %margin-vertical-2x; }
+.m_V-3x {  @extend %margin-vertical-3x; }
+.m_V-4x {  @extend %margin-vertical-4x; }
+.m_V-5x {  @extend %margin-vertical-5x; }
+.m_H-3x {  @extend %margin-horizontal-3x; }
+.m_T-2x { @extend %margin-top-2x; }
+.m_T-3x { @extend %margin-top-3x; }
+.m_T-4x { @extend %margin-top-4x; }
+.m_T-5x { @extend %margin-top-5x; }
+.m_T-6x { @extend %margin-top-6x; }
+.m_B { @extend %margin-bottom-1x; }
+.m_B-2x { @extend %margin-bottom-2x; }
+.m_B-3x { @extend %margin-bottom-3x; }
+.m_B-4x { @extend %margin-bottom-4x; }
+.m_B-5x { @extend %margin-bottom-5x; }
+.m_B-6x { @extend %margin-bottom-6x; }
+.m_L-2x { @extend %margin-left-2x; }
+.m_L-3x { @extend %margin-left-3x; }
+.m_L-4x { @extend %margin-left-4x; }
+.m_L-5x { @extend %margin-left-5x; }
+.m_L-6x { @extend %margin-left-6x; }
+.m_L-7x { @extend %margin-left-7x; }
+.m_L-8x { @extend %margin-left-8x; }
+.m_R-2x { @extend %margin-right-2x; }
+.m_R-3x { @extend %margin-right-3x; }
+.m_R-4x { @extend %margin-right-4x; }
+.m_R-5x { @extend %margin-right-5x; }
+.m_R-6x { @extend %margin-right-6x; }
+
+.m_0 {margin: 0px !important}
+.m_R-0 {margin-right: 0px !important}
+.m_T-0 {margin-top: 0px !important}
+.m_B-0 {margin-bottom: 0px !important}
+.m_auto{
+     margin-left: auto;
+     margin-right: auto;
+}
+
+.p_All-3x { @extend %padding-3x; }
+.p_All-4x { @extend %padding-4x; }
+.p_V {  @extend %padding-vertical-1x; }
+.p_V-3x {  @extend %padding-vertical-3x; }
+.p_H-3x {  @extend %padding-horizontal-3x; }
+.p_H-4x {  @extend %padding-horizontal-4x; }
+.p_T { @extend %padding-top-1x; }
+.p_T-2x { @extend %padding-top-2x; }
+.p_T-3x { @extend %padding-top-3x; }
+.p_T-4x { @extend %padding-top-4x; }
+.p_T-5x { @extend %padding-top-5x; }
+.p_T-6x { @extend %padding-top-6x; }
+.p_B-2x { @extend %padding-bottom-2x; }
+.p_B-3x { @extend %padding-bottom-3x; }
+.p_B-4x { @extend %padding-bottom-4x; }
+.p_B-5x { @extend %padding-bottom-5x; }
+.p_B-6x { @extend %padding-bottom-6x; }
+.p_L-2x { @extend %padding-left-2x; }
+.p_L-3x { @extend %padding-left-3x; }
+.p_L-4x { @extend %padding-left-4x; }
+.p_L-5x { @extend %padding-left-5x; }
+.p_L-6x { @extend %padding-left-6x; }
+.p_L-7x { @extend %padding-left-7x; }
+.p_L-8x { @extend %padding-left-8x; }
+.p_R-2x { @extend %padding-right-2x; }
+.p_R-3x { @extend %padding-right-3x; }
+.p_R-4x { @extend %padding-right-4x; }
+.p_R-5x { @extend %padding-right-5x; }
+.p_R-6x { @extend %padding-right-6x; }
+
```

Comment:
If you don't have time, no worries, curious if we can make the class names more human readable?

Putting it in user story terms: As a front-end developer, when I look at the HTML, I should be able to understand what is being accomplished by the class.